references/identity-facade/pipeline.sh

```diff
@@ -0,0 +1,356 @@
+#!/bin/sh
+# Copyright 2020 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+set -e
+
+#################################
+### function: set_idp_env_var ###
+#################################
+set_idp_env_var() {
+
+    # discovery document of an OIDC compliant IdP
+    idp_discovery_document=https://"$APIGEE_ORG"-"$APIGEE_ENV".apigee.net/v1/openid-connect/.well-known/openid-configuration
+
+    # retrieve configuration data from a discovery document
+    response=$(curl --silent -k1 -fsSL -X GET -H "Accept:application/json" "$idp_discovery_document")
```

Comment:
-s and --silent duplicate

Reply:
double check -k1 please 

-X GET is implied - so it is up to you if you want to leave for readability